xml/System.Runtime.InteropServices/ComWrappers.xml

```diff
@@ -60,12 +60,23 @@
         <Parameter Name="count" Type="System.Int32" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="obj">To be added.</param>
-        <param name="flags">To be added.</param>
-        <param name="count">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="obj">Target of the returned Vtables.</param>
+        <param name="flags">Flags used to compute Vtables.</param>
+        <param name="count">The number of elements contained in the returned memory.</param>
+        <summary>Compute the desired Vtable for <paramref name="obj" /> respecting the values of <paramref name="flags" />.</summary>
+        <returns>
+          <see cref="T:System.Runtime.InteropServices.ComWrappers.ComInterfaceEntry" /> pointer containing memory for all COM interface entries.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+All memory returned from this function must either be unmanaged memory, pinned managed memory, or have been allocated with the <xref:System.Runtime.CompilerServices.RuntimeHelpers.AllocateTypeAssociatedMemory(System.Type,System.Int32)> API.
+ 
+ If the interface entries cannot be created and a negative `count` or <code>null</code> and a non-zero `count` are returned, the call to <xref:System.Runtime.InteropServices.ComWrappers.GetOrCreateComInterfaceForObject(System.Object,System.Runtime.InteropServices.CreateComInterfaceFlags)> will throw a <xref:System.ArgumentException>.
```

Comment:
Can the tool trim the space on the left? Also, why is `<code>` in there?

Reply:
Sure, I can add the ability to trim space on the left for each line.

I'm not sure why there's a `<code>` section instead of `<see langword="null" />`. Probably got ported as-is. I can add the ability to detect those and transform them to the correct keyword.

Reply:
```suggestion
 If the interface entries cannot be created and a negative `count` or `null` and a non-zero `count` are returned, the call to <xref:System.Runtime.InteropServices.ComWrappers.GetOrCreateComInterfaceForObject(System.Object,System.Runtime.InteropServices.CreateComInterfaceFlags)> will throw a <xref:System.ArgumentException>.
```